test/unit/lib/formatter/index.js

```diff
@@ -218,6 +218,18 @@ describe('Formatter', function () {
 
       expect(reformatted).to.deep.equal(options);
     });
+
+    it('can unformat not format value', function () {
```

Comment:
can you rephrase this? I'm not sure I understand what it's testing.

Reply:
change to `should return a unformatted value`. 
Let me know if this is not very suitable / understandable.